website/project/views/register.py

```diff
@@ -301,6 +257,73 @@ def node_registration_embargo_disapprove(auth, node, token, **kwargs):
     status.push_status_message('Your disapproval has been accepted and the embargo has been cancelled.', kind='success', trust=False)
     return redirect(redirect_url)
 
+
+@must_not_be_rejected
+@must_be_valid_project
+@must_have_permission(ADMIN)
+def node_registration_approve(auth, node, token, **kwargs):
+    """Handles disapproval of registration retractions
+    :param auth: User wanting to disapprove retraction
+    :return: Redirect to registration or
+    :raises: HTTPError if invalid token or user is not admin
+    """
+
+    if not node.is_pending_registration:
+        raise HTTPError(http.BAD_REQUEST, data={
+            'message_short': 'Invalid Token',
```

Comment:
Change or remove `message_short`. Also, make sure to keep casing consistent in error messages. I believe the style guidelines recommend sentence-case.